graylog2-server/src/main/java/org/graylog2/rest/resources/cluster/ClusterSystemJobResource.java

```diff
@@ -81,8 +82,6 @@ public SystemJobSummary getJob(@ApiParam(name = "jobId", required = true) @PathP
                 if (response.isSuccessful()) {
                     // Return early because there can be only one job with the same ID in the cluster.
                     return response.body();
-                } else {
```

Comment:
As described in the commit message. :smiley: 

> Do not log an error or warning if a system job cannot be found

I think we do not have to log that case as a warning or error.